helm/chart/router/templates/deployment.yaml

```diff
@@ -109,31 +109,31 @@ spec:
               mountPath: /app/configuration.yaml
               subPath: configuration.yaml
               readOnly: true
-            {{- if index .Values.router.configuration "rhai" }}
-            - name: router-rhai
-              mountPath: {{ .Values.router.configuration.rhai.scripts | default "/dist/rhai" }}/{{ .Values.router.configuration.rhai.main | default "main.rhai" }}
-              subPath: {{ .Values.router.configuration.rhai.main | default "main.rhai" }}
-              readOnly: true
             {{- end }}
+            {{- range .Values.extraConfigmaps }}
+            - name: {{ .name }}
+              mountPath: {{ .mountPath }}
+              subPath: {{ .subPath | default "" }}
+              readOnly: {{ .readOnly }}
+            {{- end }}
+            {{- if .Values.extraVolumeMounts }}
+              {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
-          {{- if .Values.extraVolumeMounts }}
-            {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumeMounts "context" $) | nindent 12 }}
-          {{- end }}
           {{- end }}
       {{- if or .Values.router.configuration .Values.extraVolumes }}
       volumes:
         {{- if .Values.router.configuration }}
         - name: router-configuration
           configMap:
             name: {{ include "router.fullname" . }}
-        {{- if index .Values.router.configuration "rhai" }}
-        - name: router-rhai
-          configMap:
-            name: {{ include "router.rhaiConfigMapName" . }}
         {{- end }}
+        {{- range .Values.extraConfigmaps }}
+        - name: {{ .name }}
+          configMap:
+            name: {{ .name }}
         {{- end }}
         {{- if .Values.extraVolumes }}
-        {{- include "common.tplvalues.render" ( dict "value" .Values.extraVolumes "context" $) | nindent 8 }}
+          {{- include "common.tplvalues.render" ( dict "value" .Values.extraVolumes "context" $) | nindent 8 }}
```

Comment:
Was this an intended change? The spacing does matter bc of the nindent.

Reply:
The `nindent 8` at the end forces the indentation to be "new line + 8 spaces", so I was just making it a bit more readable.
I'm not a great helm templater, so I may be wrong.